pctasks/dataset/pctasks/dataset/workflow.py

```diff
@@ -19,6 +19,21 @@
 from pctasks.ingest.utils import generate_collection_json
 
 
+def task_tags(
+    task_name: str,
+    tags: Optional[Dict[str, str]],
+    task_config: Optional[Dict[str, Any]],
+) -> Optional[Dict[str, str]]:
+
+    task_config_ = {} if task_config is None else task_config
+    tags_ = {} if tags is None else tags
+    merged_tags = tags_ | task_config_.get(task_name, {}).get("tags", {})
+    if merged_tags:
+        return merged_tags
+    else:
+        return None
```

Comment:
I don't think there is any harm. A few thoughts, though:
- My original thought on returning None instead of an empty dictionary was to match the expected typing (`Optional[Dict[str, str]] = None`) for the `tags` parameter of functions consuming the output of the `task_tags` function. I'm not sure if an empty dict will pass mypy muster.
- Returning None instead of a dict when both inputs are None will protect against breaking any potential downstream `is None` checks. But I don't see any checks like that on any tags parameters, so this may be moot.

Reply:
SGMT. We can review this more thoroughly later, but this is good for now.